src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -918,25 +905,50 @@ public record SecretAddress<T>(Uri Url) : ISecret<T> {
 public record SecretData<T>(ISecret<T> Secret) {
 }
 
+[SkipRename]
+public enum JobResultType {
+    CoverageData,
+    RuntimeStats,
+}
+
 public record JobResult(
-    [PartitionKey][RowKey] Guid JobId,
+    [PartitionKey] Guid TaskId,
+    [RowKey] string MachineIdMetric,
+    Guid JobId,
     string Project,
     string Name,
-    double NewCrashingInput = 0,
-    double NoReproCrashingInput = 0,
-    double NewReport = 0,
-    double NewUniqueReport = 0,
-    double NewRegressionReport = 0,
-    double NewCrashDump = 0,
-    double InstructionsCovered = 0,
-    double TotalInstructions = 0,
-    double CoverageRate = 0,
-    double IterationCount = 0
-) : EntityBase() {
-    public JobResult(Guid JobId, string Project, string Name) : this(
-        JobId: JobId,
-        Project: Project,
-        Name: Name, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0) { }
+    string Type,
+    Dictionary<string, double> MetricValue
+) : EntityBase();
+
+public record CoverageMetricValue(IDictionary<string, double> inner) {
```

Comment:
these classes not used yet, planned for future?

Reply:
Working with Cheick to see if this is something we want to use instead of having to key into the dictionaries during the metric update in JobOperations